runelite-client/src/main/java/net/runelite/client/plugins/demonicgorilla/DemonicGorillaPlugin.java

```diff
@@ -677,7 +690,7 @@ public void onNpcDespawned(NpcDespawned event)
 	{
 		if (gorillas.remove(event.getNpc()) != null && gorillas.isEmpty())
 		{
-			clear();
+			reset();
```

Comment:
This immediately causes a npc rescan whenever the last gorilla despawns.

Reply:
I don't see much of a problem with this since it mostly only happens when leaving the gorilla area, but I guess I can put some of the old code back.